go.mod

```diff
@@ -65,4 +65,9 @@ replace (
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200204173128-addea2498afe // avoids case-insensitive import collision: "github.com/googleapis/gnostic/openapiv2" and "github.com/googleapis/gnostic/OpenAPIv2"
 )
 
+replace (
+	github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20200526122759-6061f7c8bf85
+	github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20200526123527-0616e758768b
+)
```

Comment:
reminder for removing these replace lines and for running `go mod tidy` :-)

Reply:
done in e42325c